src/videotile/DefaultVideoTile.ts

```diff
@@ -63,7 +63,20 @@ export default class DefaultVideoTile implements DevicePixelRatioObserver, Video
       // In Safari, a hidden video element can show a black screen.
       // See https://bugs.webkit.org/show_bug.cgi?id=241152 for more information.
       if (new DefaultBrowserBehavior().requiresVideoPlayWorkaround() && videoElement.paused) {
-        videoElement.play();
+        const promise = videoElement.play();
+        // See https://bugs.webkit.org/show_bug.cgi?id=243519 for more information.
+        // https://webkit.org/blog/7734/auto-play-policy-changes-for-macos/
+        /* istanbul ignore else */
+        if (promise !== undefined) {
+          promise
+            .catch(error => {
+              console.warn('Error playing video in Safari', error);
```

Comment:
Should this be an error? When this happen does the video still play?

Reply:
Yes, it plays fine both on the local and remote side, hence I created a warning, should I change that to `error`?

Reply:
If video still plays then I think warning is fine.